docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,394 @@
+PTRATIO      pupil-teacher ratio by town
+B            1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town
+LSTAT        % lower status of the population
+MEDV         Median value of owner-occupied homes in $1000’s
+============ ============================================================================
+
+The cells below show basic summary statistics about the data, the data types of the 
+columns, and the number of missing values.
+
+Note that the :func:`fairlearn.datasets.fetch_boston` function warns user by 
+default that the dataset contains fairness issues. 
+
+Setting :code:`warn=False` will turn the warning off. 
+
+To return the dataset as a :class:`pandas.DataFrame`, pass 
+:code:`as_frame=True` and call the *data* attribute.
+
+
+For more information about how to use the :code:`fetch_boston` function, 
+visit :mod:`fairlearn.datasets`. 
+
+.. note::
+
+    Calling the :func:`fairlearn.datasets.fetch_boston` function will raise a 
+    :code:`FairnessWarning`.
+    For more information on this warning refer to :mod:`fairlearn.datasets`.
+
+.. doctest:: datasets
+
+    >>> from fairlearn.datasets import fetch_boston
+    >>> import pandas as pd
+
+    >>> X, y = fetch_boston(as_frame = True, return_X_y= True)
+    >>> boston_housing = pd.concat([X, y], axis = 1)
+    >>> boston_housing.head()
+        CRIM	ZN	INDUS	CHAS	NOX	RM	AGE	DIS	RAD	TAX	PTRATIO	B	LSTAT	MEDV
+    0	0.00632	18.0	2.31	0	0.538	6.575	65.2	4.0900	1	296.0	15.3	396.90	4.98	24.0
+    1	0.02731	0.0	7.07	0	0.469	6.421	78.9	4.9671	2	242.0	17.8	396.90	9.14	21.6
+    2	0.02729	0.0	7.07	0	0.469	7.185	61.1	4.9671	2	242.0	17.8	392.83	4.03	34.7
+    3	0.03237	0.0	2.18	0	0.458	6.998	45.8	6.0622	3	222.0	18.7	394.63	2.94	33.4
+    4	0.06905	0.0	2.18	0	0.458	7.147	54.2	6.0622	3	222.0	18.7	396.90	5.33	36.2
+
+    >>> boston_housing.describe()
+        CRIM	ZN	INDUS	NOX	RM	AGE	DIS	TAX	PTRATIO	B	LSTAT	MEDV
+    count	506.000000	506.000000	506.000000	506.000000	506.000000	506.000000	506.000000	506.000000	506.000000	506.000000	506.000000	506.000000
+    mean	3.613524	11.363636	11.136779	0.554695	6.284634	68.574901	3.795043	408.237154	18.455534	356.674032	12.653063	22.532806
+    std	8.601545	23.322453	6.860353	0.115878	0.702617	28.148861	2.105710	168.537116	2.164946	91.294864	7.141062	9.197104
+    min	0.006320	0.000000	0.460000	0.385000	3.561000	2.900000	1.129600	187.000000	12.600000	0.320000	1.730000	5.000000
+    25%	0.082045	0.000000	5.190000	0.449000	5.885500	45.025000	2.100175	279.000000	17.400000	375.377500	6.950000	17.025000
+    50%	0.256510	0.000000	9.690000	0.538000	6.208500	77.500000	3.207450	330.000000	19.050000	391.440000	11.360000	21.200000
+    75%	3.677083	12.500000	18.100000	0.624000	6.623500	94.075000	5.188425	666.000000	20.200000	396.225000	16.955000	25.000000
+    max	88.976200	100.000000	27.740000	0.871000	8.780000	100.000000	12.126500	711.000000	22.000000	396.900000	37.970000	50.000000    
+
+.. _dataset_issues:
```

Comment:
same for this marker: `_boston_dataset_issues`

Reply:
Done